src/pages/console/formbuilder/data-binding.mdx

```diff
@@ -0,0 +1,78 @@
+export const meta = {
+  title: 'Data binding',
+  description:
+    'Cloud connected forms can be bound to data models with relationships, allowing multiple data models to be updated upon submission.'
+};
+
+## Connected forms
+Connected Forms are bound to a model in your data schema. Whenever a connected form is submitted, a record is created or updated in the bound data model automatically, with some or all of the form's input fields mapping to fields in the data model. Connected forms work automatically - no `onSubmit` handling is required.
+
+## Unconnected forms
```

Comment:
unconnected forms read awkwardly to me

Reply:
@abdallahshaban557 - do you have any recommendations? I actually had these forms labeled as "ad hoc" forms, but that felt even less clear. "Unconnected forms" is awkward, but at least it's clear.